.drevops/tests/bats/notify.bats

```diff
@@ -8,48 +8,75 @@ load _helper.bash
 
 @test "Notify: skip" {
   pushd "${LOCAL_REPO_DIR}" >/dev/null || exit 1
+  declare -a STEPS=(
```

Comment:
@richardgaunt 
Could you please revert this. I should have been more clear: we need to only convert tests that are using mocks. This one does not use mocks so it is easier to maintain it as normal asserts. sorry